### PR TITLE
chore: Update sidetree-core revision

### LINKED ISSUE #293: Update sidetree-core revision

### PR DESCRIPTION
**Pull request by fqutishat**
closes #293

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/restapi/vc/operation/operations.go [hunk 1]
```diff
@@ -697,6 +695,75 @@ func (o *Operation) retrieveCredentialHandler(rw http.ResponseWriter, req *http.
...
+	}
+
+	opts = append(opts,
+		uniregistrar.WithPublicKey(&didmethodoperation.PublicKey{
+			ID: pubKey1, Type: didclient.JWSVerificationKey2020,
+			Value:    base64.StdEncoding.EncodeToString(base58.Decode(base58PubKey)),
+			Encoding: didclient.PublicKeyEncodingJwk, Usage: []string{didclient.KeyUsageGeneral, didclient.KeyUsageOps}}),
+		uniregistrar.WithPublicKey(&didmethodoperation.PublicKey{
+			ID: recoveryKey1, Type: didclient.JWSVerificationKey2020,
+			Value:    base64.StdEncoding.EncodeToString(base58.Decode(recoveryPubKey)),
```
Comment (troyronda) [inline pkg/restapi/vc/operation/operations.go:718]: we still need to have an ed25519 signature suite key available to sign VCs for the interop requirements.

Comment (fqutishat) [reply pkg/restapi/vc/operation/operations.go:718]: yes

Comment (fqutishat) [reply pkg/restapi/vc/operation/operations.go:718]: I will add it in next update for sidetree

File: test/bdd/testdata/profile_request_template.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "",
   "uri": "https://example.com/credentials",
-  "signatureType": "Ed25519Signature2018",
```
Comment (sandrask) [inline test/bdd/testdata/profile_request_template.json:4]: No signature type?

Comment (fqutishat) [reply test/bdd/testdata/profile_request_template.json:4]: yes

Comment (fqutishat) [reply test/bdd/testdata/profile_request_template.json:4]: configure from bdd test

Comment (fqutishat) [reply test/bdd/testdata/profile_request_template.json:4]: https://github.com/trustbloc/edge-service/pull/296/files#diff-05fd512fb881602f4ac8d3b2d3a6c694R19

**Pull request merged**